projects/collection-editor-library/src/lib/components/meta-form/meta-form.component.ts

```diff
@@ -32,11 +32,12 @@ export class MetaFormComponent implements OnChanges, OnDestroy {
   public previousShuffleValue: boolean;
   public subscription: Subscription;
   public formSectionProperties:any;
+  public formCriteriaSection:any[]=[];
```

Comment:
@sohailamjad12 Rename formCriteriaSection to more relevant and generic name -> questionCountCriteriaFields